client/python/requirements.txt

```diff
@@ -1,4 +1,4 @@
-astroid==2.5
+astroid==2.11
```

Comment:
```
ERROR: Cannot install -r client/python/requirements.txt (line 24) and astroid==2.5 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested astroid==2.5
    pylint 2.13.0 depends on astroid<=2.12.0-dev0 and >=2.11.0

```